src/auth.js

```diff
@@ -0,0 +1,51 @@
+import jsonwebtoken from "jsonwebtoken";
+
+const USERNAME = "user";
+const PASSWORD = "password";
+const tokens = [];
+const JWT_SECRET = "blahblahblahblahBLAHBLAHBLAHBLAHBLAHBLAHBLAHBLAHBLAAAA";
+const FIVE_MINS = 5 * 60 * 1000;
+
+export const login = (req, res) => {
+	const authHeader = req.headers.authentication;
+  const b64creds = authHeader.slice(6);
+  const bufferedInput = Buffer.from(b64creds, "base64")
+  const creds = bufferedInput.toString();
+	const [user, password] = creds.split(":");
+  
+  if (user == USERNAME && password == PASSWORD) {
+		const jwt = jsonwebtoken.sign(
+			{
+				username: user
+			},
+			JWT_SECRET,
+			{ expiresIn: FIVE_MINS } 
+		);
+		tokens.push(jwt);
```

Comment:
This is not necessary (and the `tokens` array is not used anywhere). JWTs are designed to work statelessly, i.e. without having to store the tokens, because they can be decoded and signature-verified every time the client sends them, without having to look them up anywhere.